standards/application/webrtc.md

```diff
@@ -0,0 +1,331 @@
+  - **For `OFFER`**: this field contains the WebRTC SDP Offer generated by the initiator (`RTCPeerConnection.createOffer()`), encoded as bytes. Typically this is the UTF-8 encoded SDP text (it **MAY** be compressed or binary encoded if both sides agree, but plain SDP text is simplest).
+  - **For `ANSWER`**: this field contains the WebRTC SDP Answer generated by the responder (`RTCPeerConnection.createAnswer()`), encoded in the same manner as the offer.
+  - **For `CANDIDATE`**: this field contains an ICE candidate to be added. The candidate can be encoded as the JSON or string representation of the ICE candidate (as returned by the WebRTC API). Each `CANDIDATE` message **SHOULD** carry a single ICE candidate. (Multiple `CANDIDATE` messages will be sent as new candidates are gathered.) Both peers may send `CANDIDATE` messages after sending or receiving an Offer/Answer, as they gather ICE candidates.
+  - **For `REJECT`, `BUSY`, and `END`**: this field is unused and **SHOULD** be omitted (or empty). These message types carry no additional data beyond what is implied by their type.
+
+All multi-byte fields (IDs, SDP, etc.) are transmitted as bytes. Text values like SDP can be UTF-8 encoded to bytes. Peers are responsible for serializing and deserializing the SDP and ICE data to/from this message format.
+
+Because messages are sent via Waku, the `WakuWebRTCMessage` will itself be the payload of a Waku envelope (`WakuMessage`). That outer envelope’s `contentTopic` must be set appropriately (as discussed above), and the payload **SHOULD** be encrypted so that only the intended recipient can decode it. Waku supports both symmetric and asymmetric payload encryption. For example, if the initiator knows the responder’s static public key (from their peer ID), the Offer could be asymmetrically encrypted for that key so only the responder can decrypt it. Alternatively, the application can establish a symmetric key (out-of-band or via a separate handshake) and use that to encrypt all signaling messages for that session. At a minimum, messages **SHOULD** be encrypted using Waku Message Version 1 (which provides Whisper-style symmetric or asymmetric encryption). This ensures that even if an adversary observes the content topic or intercepts messages, they cannot read sensitive information like IP addresses in ICE candidates or session descriptions. (If no encryption key is provided, the signaling messages would be sent in plaintext, which is **NOT** recommended.)
+
+## Signaling Flow
+
+This section describes the typical flow of messages to establish and terminate a WebRTC connection between two peers. The protocol is symmetric in that either peer can play the initiator role at any time to start a new session. Implementations should be prepared to handle incoming connection offers while not interfering with any ongoing sessions (e.g., if a peer is already in a session, it might reject new incoming offers or signal busy).
```

Comment:
It's been already mention that any peer can be the initiator, let's avoid redundant information in specs.